src/main/webapp/queue.js

```diff
@@ -44,6 +65,44 @@ function updateShareTab() {
       'lounge.html/?room_id='+ roomParameters +'" type="text" readonly</input>';
 }
 
+/* exported voteToSkip */
+// Tracks the user's choice to skip a video and alters firestore as needed
+function voteToSkip() {
+  voteBtnCount++;
+  if (voteBtnCount % 2 == 1) {
+    db.collection('rooms') // eslint-disable-line no-undef
+        .doc(roomParameters)
+        .collection('CurrentVideo')
+        .doc('PlaybackData')
+        .update({
+          skipVotes: firebase // eslint-disable-line no-undef
+              .firestore.FieldValue.increment(1),
+        }).then(function() {
+          console.log('votes incremented');
+        }).catch(function(error) {
+          console.log('voting caused an error: ', error);
+        });
+  } else if (voteBtnCount % 2 == 0) {
+    db.collection('rooms') // eslint-disable-line no-undef
```

Comment:
seems like the logic in if and else if is similar only for increment and decrement. Move this to a method and pass increment or decrement parameters instead.